pybind11-BUILD.bazel

```diff
@@ -16,23 +16,7 @@ cc_library(
             "include/pybind11/common.h",
         ],
     ),
-    copts = select({
-        ":msvc_compiler": [],
-        "//conditions:default": [
-            "-fexceptions",
-            # Useless warnings
-            "-Xclang-only=-Wno-undefined-inline",
```

Comment:
I have never seen this style of flags before. Maybe these are specific to Google's internal complete wrapper and required to get `parse_headers` to pass with it?